.github/workflows/publish-release.yml

```diff
@@ -33,20 +33,23 @@ jobs:
       - uses: actions/cache@v3
         id: restore-build
         with:
-          path: |
-            ./*
-            !./node_modules/
+          path: ./dist
           key: ${{ github.sha }}
 
   publish-npm:
     environment: publish
     runs-on: ubuntu-latest
     needs: publish-release
     steps:
+      - uses: actions/checkout@v2
+        with:
+          # We check out the release pull request's base branch, which will be
+          # used as the base branch for all git operations.
+          ref: ${{ github.event.pull_request.base.ref }}
```

Comment:
To be honest I am not sure what happens if you use `.ref` and another commit gets pushed before the action is run :thinking: It might only checkout from the repo in the state at the time the event was triggered. But I guess it couldn't hurt to prefer `sha` if we're not sure.